src/Containers/Microsoft.NET.Build.Containers/Tasks/ComputeDotnetBaseImageAndTag.cs

```diff
@@ -105,10 +105,23 @@ public override bool Execute()
         return !Log.HasLoggedErrors;
     }
 
+    private string UbuntuCodenameForSDKVersion(SemanticVersion version)
```

Comment:
If this only goes into the 8.0.300+ SDK, how would we get anything less than 8.0.300 here? I'm assuming we aren't planning to backport this to earlier versions?

Reply:
This logic also ships in the NuGet package for containerization, which is useful for all kinds of users not on the 8.0.200+ set of SDKs. It happens to be compatible with downlevel SDKs all the way back to 7.0.100 so we've kept publishing it. Now that 8.0.200 has 'completed' the merge of the feature into the SDK we'll likely want to remove the package for 9.0.100, but until we do that we should maintain compat and expect to be used by older SDKs.